src/components/FilterGroup.tsx

```diff
@@ -94,6 +100,11 @@ export function FilterGroup({
             />
           );
         })}
+        {limited &&
+        /* eslint-disable-next-line react-perf/jsx-no-new-function-as-prop */
```

Comment:
I think it's a little nicer if we indent everything after the && in this conditional (lines 104-107) and move the last brace to the next line